### ISSUE TITLE
importnb failing on the nightly build

### ISSUE THREAD

**Issue by tonyfast** [deathbeds/importnb#81]
* https://travis-ci.org/deathbeds/importnb/jobs/458196784
* https://github.com/python/cpython/commit/c93c58b5d560cfe44d9884ff02c9b18e06333360#diff-90b16efb966d69b1b16064eb8cf37550 removes `_installed_safely`. 

 `importnb` needs to work around this change.

**Comment by bollwyvl** [comment]
This will probably come to a head here:
https://github.com/conda-forge/importnb-feedstock/pull/17

**Comment by bollwyvl** [comment]
Hooray having fixed this for 3.8!

Boo, broken again for py3.9:

https://travis-ci.org/deathbeds/importnb/jobs/636384742#L795

```
~/build/deathbeds/importnb/src/importnb/loader.py in <module>

     37     from importlib._bootstrap import _load_unlocked, _requires_builtin

     38 else:

---> 39     from importlib._bootstrap import _installed_safely, _requires_builtin

     40 

     41 

ImportError: cannot import name '_installed_safely' from 'importlib._bootstrap' (/home/travis/build/deathbeds/importnb/.tox/ipython/lib/python3.9/importlib/_bootstrap.py)

ERROR: InvocationError for command /home/travis/build/deathbeds/importnb/.tox/ipython/bin/ipython setup.py test (exited with code 1)
```

**Comment by bollwyvl** [comment]
Think we're good here now... until we start tracking 3.10 :tired_face:
